test/index.js

```diff
@@ -79,6 +79,19 @@ describe('Metal Tools - Soy', function() {
         done();
       });
     });
+
+    it('should use custom handleError function', function(done) {
+      var stream = metalToolsSoy({
+        dest: 'test/fixtures/soy',
+        handleError: function(err) {
+          assert.ok(err);
+          assert.ok(err.message);
+
+          done();
+        },
+        src: 'test/fixtures/soy/compileError.soy'
+      });
```

Comment:
Due to the compile error the `end` event won't actually fire, unless I manually emitted it from the `handleError` function. So either way I won't be able to stub it.